articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+<!--more-->
+CSS is the main playground for any developer. To use an analogy, CSS is like all the spices you add to your food to give it a rich, enticing aroma and the desired taste when you finally present it. However, just throwing things in there randomly doesn't guarantee greatness.
+
+You still have to know what goes with what, when to apply a specific style or when something just doesn't fit in. How you implement your CSS can either make or break your website. Below we'll be looking at the different styling options in CSS, how to apply them, and when best to implement each style.
+
+### Styling in CSS
+There are 3 distinct methods for styling in CSS. Each level of styling is given a different hierarchical priority and is used for different reasons.
+
+The 3 methods are further grouped into two categories. Namely *Internal CSS* and *External CSS*. *Internal CSS* is comprised of Local Styles and Page-Level Styles while *External CSS* is just as the name implies.
+
+1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+
+```HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+        <p>This
+    </body>
+  </html>
+```
+
+By using the style tag in the HTML element *p*, we're able to change the font, font-size, and style only for the first *p*. As for the second *p* element it retains its default style.
+
+2. **Page-Level styles:** Page-level styles are defined at the header area of the HTML file. All similar tags, whether elements members of the class or ID selector within the body of the HTML will undergo the changes at once.
+
+```HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+      <meta charset="utf-8">
+      <style type="text/css">
+         body{
+           color: yellow;
+           background-color: red;
+         }
+         p{
+           color: red;
+           background-color: yellow;
+         }
+      </style>
+    </head>
+    <body>
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+```
+
+This time around we defined the style within the *head* HTML tags. By doing so all elements defined in the style will automatically use the style defined for them. This means that all *p* elements on the page will have a color red and a background-color of yellow. The body on the other hand will have the color yellow and a background-color of red.
+
+3. **External Styles:** The styles used for the webpage are located in a completely different file. The *.HTML* file is linked to the *.css* file that can be imported to modify the style. When developing a website with multiple pages this styling usually comes in handy.
```

Comment:
This sentence is a bit vague - not very easy to follow along. 
Please break it down and elaborate. 
A different file? is this the HTML file you talk about that is linked?
Then you jump into a code snippet below with no context as to what its for or how its used.